server/aws-lsp-codewhisperer/src/language-server/codeWhispererServer.ts

```diff
@@ -191,6 +202,9 @@ const emitAggregatedUserTriggerDecisionTelemetry = (
             : undefined,
         codewhispererTimeToFirstRecommendation: session.timeToFirstRecommendation,
         codewhispererPreviousSuggestionState: session.previousTriggerDecision,
+        codewhispererSupplementalContextTimeout: session.supplementalMetadata?.isProcessTimeout,
+        codewhispererSupplementalContextIsUtg: session.supplementalMetadata?.isUtg,
+        codewhispererSupplementalContextLength: session.supplementalMetadata?.contentsLength,
```

Comment:
Is it expected we set 3 props in this metric, but 4 above? The tests are checking for 4 properties, at least.